irohad/ametsuchi/impl/postgres_block_index.cpp

```diff
@@ -5,125 +5,127 @@
 
 #include "ametsuchi/impl/postgres_block_index.hpp"
 
-#include <unordered_set>
+#include <numeric>
 
-#include <boost/range/adaptor/filtered.hpp>
+#include <boost/format.hpp>
 #include <boost/range/adaptor/indexed.hpp>
-#include <boost/range/algorithm/for_each.hpp>
-#include <boost/range/numeric.hpp>
 #include "common/types.hpp"
 #include "common/visitor.hpp"
-#include "cryptography/ed25519_sha3_impl/internal/sha3_hash.hpp"
 #include "interfaces/commands/command.hpp"
 #include "interfaces/commands/transfer_asset.hpp"
+#include "interfaces/common_objects/types.hpp"
 #include "interfaces/iroha_internal/block.hpp"
 
-namespace iroha {
-  namespace ametsuchi {
+namespace {
+  // Return transfer asset if command contains it
+  boost::optional<const shared_model::interface::TransferAsset &>
+  getTransferAsset(const shared_model::interface::Command &cmd) noexcept {
+    using ReturnType =
+        boost::optional<const shared_model::interface::TransferAsset &>;
+    return iroha::visit_in_place(
+        cmd.get(),
+        [](const shared_model::interface::TransferAsset &c) {
+          return ReturnType(c);
+        },
+        [&](const auto &command) -> ReturnType { return boost::none; });
+  }
 
-    bool execute(soci::statement &st) {
-      st.define_and_bind();
-      try {
-        st.execute(true);
-        return true;
-      } catch (const std::exception &e) {
-        return false;
-      }
-    }
+  // tx hash -> block where hash is stored
```

Comment:
Very strange comment, did you want to add docs for a function?

Reply:
As I see, all methods in this namespace have the same comments, maybe change them to usual docs format?

Reply:
They are private methods of the file. I do not see the point of adding doc comments for since they are not supposed to be used outside. I will add more descriptive comments though